x-pack/plugins/apm/public/components/app/service_inventory/no_services_message.tsx

```diff
@@ -23,10 +22,6 @@ interface Props {
 export function NoServicesMessage({ historicalDataFound, status }: Props) {
   const upgradeAssistantHref = useUpgradeAssistantHref();
 
-  if (status === 'loading') {
-    return <LoadingStatePrompt />;
-  }
```

Comment:
Is this no longer needed?

Reply:
not really, the loading state is going to be handled by the EuiBasicTable now, https://github.com/elastic/kibana/pull/107094/files#diff-5eecffd3503b82b7d3d75728d267e8e1f3a98f9b2228a0f7a86c848c4d286f83R130